client/hotrod-client/src/main/java/org/infinispan/client/hotrod/RemoteCacheManager.java

```diff
@@ -658,6 +658,7 @@ private Properties loadFromStream(InputStream stream) {
                   return null;
                }
             }
+            result.resolveCompatibility();
```

Comment:
Since resolveCompatibility() actually pings, this makes pingOnStartup() essentially redundant ?